superset-frontend/src/types/bootstrapTypes.ts

```diff
@@ -150,6 +151,7 @@ export interface CommonBootstrapData {
   extra_sequential_color_schemes: SequentialSchemeConfig[];
   theme_overrides: JsonObject;
   menu_data: MenuData;
+  d3_format: Partial<FormatLocaleDefinition>;
```

Comment:
Do we really need `Partial` here? It appears that the added `dict` in `config.py` defines all properties of `FormatLocaleDefinition`. If not, let's make sure the type in `config.py` reflects the partialness.